jaeger-core/src/main/java/com/uber/jaeger/senders/HttpSender.java

```diff
@@ -44,7 +46,7 @@
   private static final TProtocolFactory PROTOCOL_FACTORY = new Factory();
   private static final TSerializer SERIALIZER = new TSerializer(PROTOCOL_FACTORY);
 
-  private final HttpClient httpClient = new DefaultHttpClient();
+  private final HttpClient httpClient = new DefaultHttpClient(new PoolingClientConnectionManager());
```

Comment:
@yurishkuro okhttp looks good, and a smaller size helps especially if we need to shade it.

@pavolloffay I think so as well, but I think moving to separate modules can wait until we start supporting transports other than http/udp